index.js

```diff
@@ -15,7 +15,7 @@ const isProduction = process.env.NODE_ENV === 'production';
 const app = express();
 
 app.use(cors());
-app.use(express.urlencoded({ extended: true }));
+app.use(express.urlencoded({ extended: false }));
```

Comment:
Why you choose to disable it, and you leave it there? (Why disabling it?)

Reply:
Thank you @niyobobo I forgot to change it, but now it is solved.